vendor/k8s.io/kubernetes/openshift-kube-apiserver/admission/scheduler/podtolerations/admission.go

```diff
@@ -0,0 +1,114 @@
+package podtolerations
+
+import (
+	"fmt"
+	"io"
+
+	"k8s.io/apiserver/pkg/admission"
+	"k8s.io/apiserver/pkg/admission/initializer"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+	coreapi "k8s.io/kubernetes/pkg/apis/core"
+)
+
+const (
+	PluginName       = "PodTolerations"
```

Comment:
this should be something int he openshift namespace.  `scheduling.openshift.io/RestrictPodTolerations`